swarm/network/kademlia_test.go

```diff
@@ -241,16 +238,152 @@ func (tk *testKademlia) checkHealth(expectHealthy bool, expectSaturation bool) {
 	pp := NewPeerPotMap(tk.NeighbourhoodSize, addrs)
 	healthParams := tk.Healthy(pp[kid])
 
-	// definition of health, all conditions but be true:
-	// - we at least know one peer
-	// - we know all neighbors
-	// - we are connected to all known neighbors
```

Comment:
Why is the comment gone?

Reply:
Moved to the `IsHealthyStrict()` function